test/perf/perf_fast_forward.cc

```diff
@@ -40,6 +40,7 @@
 #include "sstables/compaction_manager.hh"
 #include "transport/messages/result_message.hh"
 #include "sstables/shared_index_lists.hh"
+#include <fstream>
```

Comment:
Commit message is wrong. It says `<stream>` instead of `<fstream>`

Reply:
Fixed.